docs/.vitepress/config.ts

```diff
@@ -0,0 +1,51 @@
+import { defineConfig } from 'vitepress'
+import { resolve } from 'path'
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: ' ',
+  description: "Remsfal Dokumentation",
+  lang: 'de-DE',
+  outDir: resolve(__dirname, '../../dist/docs'),
+  base: '/docs',
+  themeConfig: {
+    logo: 'logo.png',
+    // https://vitepress.dev/reference/default-theme-config
+    nav: [
+      { text: 'Forschung', link: 'https://remsfal.de/research', target: '_self'},
+      { text: 'Dokumentation', link: 'https://remsfal.de/research', target: '_self'},
+      { text: 'Kostenlos testen', link: 'https://remsfal.online/api/v1/authentication/login'}
```

Comment:
Just a link to the online domain is enough.

```suggestion
      { text: 'Kostenlos testen', link: 'https://remsfal.online'}
```